data/hai ships.txt

```diff
@@ -300,12 +315,31 @@ ship "Shield Beetle" "Shield Beetle (Pulse)"
 	gun "Pulse Cannon"
 	gun "Pulse Cannon"
 
+ship "Shield Beetle" "Shield Beetle (Tracker)"
+	outfits
+		"Hai Tracker Pod" 8
+		"Hai Tracker" 560
+		"Tracker Storage Pod" 4
+		"Pulse Turret" 4
+		
+		"Boulder Reactor"
+		"Geode Reactor"
+		"Hai Valley Batteries"
+		"Mass Expansion" 2
```

Comment:
added the 2 expansions to fit 4 storages, I can take 1 out to half the storages it has?